fe/fe-core/src/main/java/com/starrocks/qe/ShowResultSet.java

```diff
@@ -129,9 +130,8 @@ public TShowResultSet tothrift() {
         }
 
         set.resultRows = Lists.newArrayList();
-        for (int i = 0; i < resultRows.size(); i++) {
-            ArrayList<String> list = Lists.newArrayList();
-            list.addAll(resultRows.get(i));
+        for (List<String> resultRow : resultRows) {
+            List<String> list = resultRow.stream().map(x -> Objects.toString(x, "")).collect(Collectors.toList());
```

Comment:
why resultRows.get(i) will be null?

Reply:
There's no explicit limitation for null.

For `show materialized views` command, the `inactive_reason` field may be null.